actors/miner/src/deadline_state.rs

```diff
@@ -22,9 +22,11 @@ use super::{
     BitFieldQueue, ExpirationSet, Partition, PartitionSectorMap, PoStPartition, PowerPair,
     SectorOnChainInfo, Sectors, TerminationResult,
 };
+use crate::SECTORS_AMT_BITWIDTH;
 
 // Bitwidth of AMTs determined empirically from mutation patterns and projections of mainnet data.
-const DEADLINE_PARTITIONS_AMT_BITWIDTH: u32 = 3; // Usually a small array
+const DEADLINE_PARTITIONS_AMT_BITWIDTH: u32 = 3;
+// Usually a small array
```

Comment:
Either:

1. Don't wrap.
2. Move the comment up.

Reply:
rustfmt did it i am innocent